Doc/Zsh/builtins.yo

```diff
@@ -768,7 +768,7 @@ However, if the current event has added entries to the history with
 includes all new history entries since the current event began.
 
 When the tt(-l) flag is given, the resulting events are listed on
-standard output.  Otherwise the editor program var(ename) is invoked on a
+standard output.  Otherwise the editor program specified by tt(-e) var(ename) is invoked on a
```

Comment:
I suggest to add commas around `var(ename)`.

I assume the point of the change is to add the string «-e» somewhere in the help so it can more easily be grepped for.  +1 to that.

Reply:
That's exactly the goal :)

Reply:
I like it better without the commas, so that "-e ename" appears in the text the same way it does in the summary at the top of the section.

Reply:
I think the grammar are needed for syntax reasons (English syntax, not yodl). The greppability goal is achieved by adding the option name.

Reply:
I'm used to discussing "the `-e` option", not *"the `-e` _ename_ option". Furthermore, I don't generally expect synopsis and prose to use precisely the same words and markup.

All that said, I'll merge whatever you write so long as it's correct syntax. I don't overrule patch submitters :-)

Reply:
I don't think it is incorrect syntax as it stands.  It certainly reads naturally to me.

Reply:
Fair enough. Merged in 63cc5279b38bb07acd8074dcfcd0b0c354e3dfc9. Cheers!